pkg/debug/checkups/osquery.go

```diff
@@ -61,26 +60,19 @@ func (o *osqueryCheckup) version(ctx context.Context) (string, error) {
 }
 
 func (o *osqueryCheckup) interactive(ctx context.Context) error {
-	var launcherPath string
-	switch runtime.GOOS {
-	case "linux", "darwin":
-		launcherPath = "/usr/local/kolide-k2/bin/launcher"
-	case "windows":
-		launcherPath = `C:\Program Files\Kolide\Launcher-kolide-k2\bin\launcher.exe`
-	}
-
 	cmdCtx, cmdCancel := context.WithTimeout(ctx, 20*time.Second)
 	defer cmdCancel()
 
-	cmd := exec.CommandContext(cmdCtx, launcherPath, "interactive")
+	osqPath := o.k.LatestOsquerydPath(ctx)
+	cmd := exec.CommandContext(cmdCtx, osqPath, "-S")
```

Comment:
The point of running `launcher interactive` instead of osquery directly was to test (and time) the launcher <=> osquery communication piece -- I'm not sure we want to change this, given that?

Reply:
when I had discussed taking this idea to use for the osq_data check @directionless had pointed out it would be preferable to invoke osq interactive directly (vs having launcher flare invoke launcher interactive and so on). I had agreed this seemed more direct but likely overstepped updating this existing check as well- any preference here @directionless?

Reply:
discussed offline, moving this back to the original `launcher interactive` call to preserve the original intent